src/main/java/gregtech/common/metatileentities/steam/boiler/SteamCoalBoiler.java

```diff
@@ -38,7 +38,7 @@ public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
 
     @Override
     protected int getBaseSteamOutput() {
-        return isHighPressure ? 300 : 120;
+        return isHighPressure ? 15 : 6;
```

Comment:
Why were these changes made? This affects the amount of steam produced by the boiler, not just the tooltip